e2e/tests/localAuthorityMaintainsCaseAfterSDO_test.js

```diff
@@ -57,9 +57,12 @@ Before(async (I, caseViewPage, submitApplicationEventPage, sendCaseToGatekeeperE
 Scenario('local authority creates CMO', async (I, caseViewPage, draftCaseManagementOrderEventPage) => {
   await caseViewPage.goToNewActions(config.applicationActions.draftCaseManagementOrder);
   await draftCaseManagementOrderEventPage.associateHearingDate('1 Jan 2050');
+  I.click('Continue');
+  await draftCaseManagementOrderEventPage.enterDatesForDirections(directions[0]);
   await I.completeEvent('Submit');
   caseViewPage.selectTab(caseViewPage.tabs.draftOrders);
   I.seeAnswerInTab(1, 'Case management order', 'Which hearing is this order for?', '1 Jan 2050');
+  I.seeAnswerInTab(2, 'Directions 1', 'Due date and time', '1 Jan 2050, 12:00:00 PM');
```

Comment:
Are we happy with just checkign dates? no the title / body

Reply:
I think so, service and int test do a better job of testing it's assigned correctly.